### PR TITLE
[Operator] Add softmax support 

### PR DESCRIPTION
**Pull request by jroesch**
Got everything to build, and did a quick bit of hacking in between working on my talk, let me know if there is anything else to do.

### CODE REVIEW

File: quick_test.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import torch
```
Comment (bwasti) [inline quick_test.py]: perhaps we can delete this file? looks like it is well tested in test_operators.py

File: test/test_operators.py [hunk 2]
```diff
@@ -286,6 +286,23 @@ def reshape(input):
         ref_out, tvm_out = self.runBoth(reshape, input)
         assert torch.allclose(ref_out, tvm_out, rtol=0.01, atol=0.01)
 
+    @TVMTest.given(
+        shape=TVMTest.rand_shape(rank=2, min_dim=4),
```
Comment (bwasti) [inline test/test_operators.py]: nit: `axis=TVMTest.rand_int(0, 2),`

Comment (jroesch) [reply test/test_operators.py]: I agree much better to have randomized inputs 👍

File: test/test_operators.py [hunk 3]
```diff
@@ -286,6 +286,18 @@ def reshape(input):
...
+        axis=TVMTest.rand_int(0, 2),
```
Comment (bwasti) [inline test/test_operators.py]: whoops, I guess it's supposed to be `rand_int(0, 1)`
(https://circleci.com/gh/pytorch/tvm/428?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)
